src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,97 @@
-export const Pagination = () => {};
+import React, { FunctionComponent } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import classNames from 'classnames';
+
+type Props = {
+  itemsOnPage: number[]
+  currentPage: number;
+  totalPages: number[];
+  onPageChange: (event: React.MouseEvent<HTMLElement>) => void;
+};
+
+export const Pagination: FunctionComponent<Props> = ({
+  itemsOnPage,
+  currentPage,
+  totalPages,
+  onPageChange,
+}) => {
+  // const isActivePage = (num: number): boolean => num === currentPage;
+  const isPrevPage = currentPage === 1;
+  const isNextPage = currentPage === totalPages.length;
+
+  return (
+    <>
```

Comment:
react fragment is redundant here

Reply:
In this case, we have two ul elements with the same nesting, so I can't avoid react fragment usage.